packages/Debian/x86_64/DEBIAN/control

```diff
@@ -3,7 +3,11 @@ Version: 1.0.0
 Section: devel
 Priority: optional
 Architecture: amd64
-Depends: autoconf,curl,libtool,libboost-dev,libboost-system-dev,libboost-thread-dev,libpq-dev,python3-dev,python3-pip,python3-setuptools,sqlite3,sudo,krb5-user,libcurl4-openssl-dev,python3-numpy
+Build-Depends: autoconf, libtool, libboost-dev, libboost-system-dev,
+ libboost-thread-dev, libpq-dev, python3-dev, libcurl4-openssl-dev
+Depends: curl, libboost-system1.71.0, libboost-thread1.71.0,
```

Comment:
`libboost-system1.71.0 and libboost-thread1.71.0` are versioned package names which might not exactly match the package names available in all Ubuntu versions.

Reply:
Fixed in PR #112